native/package.json

```diff
@@ -1,9 +1,41 @@
 {
-  "private": true,
   "name": "native",
-  "version": "0.0.0",
-  "description": "This package.json scopes all native packages to react@^16.0.0.",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "ios": "node node_modules/react-native/local-cli/cli.js run-ios",
+    "android": "node node_modules/react-native/local-cli/cli.js run-android",
+    "test": "jest",
+    "storybook": "storybook start -p 7007"
+  },
+  "dependencies": {
+    "react": "16.0.0-alpha.12",
+    "react-native": "0.47.2"
```

Comment:
we need to make sure these are the same as what's used in the internal iOS and android apps.

Reply:
Plus I reckon they can be `devDependencies`.

Reply:
Just looked at the iOS and Android apps, iOS matches this, Android is on `16.0.0-alpha.8`. I reckon therefore we can stick with this. The version of RN is a bit less clear cut due to the fork, but I understand that this is outside of the scope of this PR.